samples/bluetooth/nrf_dm/README.rst

```diff
@@ -0,0 +1,188 @@
+.. _ble_nrf_dm:
+
+Bluetooth: nRF Distance Measurement with Bluetooth LE discovery
+###############################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Distance Measurement sample demonstrates the functionality of the Distance Measurement (DM) subsystem.
+It shows how to use DM to measure the distance between devices.
+The Bluetooth Heart Rate Service is running simultaneously.
+
+Overview
+********
+
+The sample initializes and performs distance measurements between devices.
+The procedure for distance measurement on both devices must be synchronized.
+
+The synchronisation step has two goals:
+
+   * Detect peers that support the ranging procedure.
+   * Provide synchronisation for the ranging step.
+
+This application synchronizes devices using the scan and advertising facilities of the Bluetooth LE.
+Anchoring occurs after the exchange of ``SCAN_REQ`` and ``SCAN_RSP`` packets.
+
+.. figure:: images/sync.svg
+   :alt: Synchronisation process
+
+   Synchronisation process
+
+Both devices act as advertiser and scanner.
+Device B receives an advertising from Device A and scans Device A.
+Device A sends a scan response which contains ``Manufacturer Specific Data``.
+This message contains the following fields:
+
+   * Company Identifier Code.
+   * Identifier indicating that the device supports distance measurement.
+   * The access address used for measurement.
+
+This scanner/advertiser interaction is used as a synchronisation point.
+After a configurable delay, Device A and Device B try to range each other.
+
+   * If a Device has acted as an advertiser in synchronization, it will act as the reflector in the ranging procedure.
+   * If a Device has acted as a scanner in synchronization, it will act as the initiator in the ranging procedure.
```

Comment:
```suggestion
   * If a Device acts as an advertiser in synchronization, it will act as the reflector in the ranging procedure.
   * If a Device acts as a scanner in synchronization, it will act as the initiator in the ranging procedure.
```